src/witan/gwyn/schemas.clj

```diff
@@ -31,7 +32,7 @@
 
 (def HistoricalFireRiskScores
   (make-ordered-ds-schema [[:address s/Str] [:name s/Str] [:risk-score java.lang.Double]
-                           [:date-last-risk-assessed s/Any]]))
+                           [:date-last-risk-assessed sc/ISO-Date-Time]]))
```

Comment:
@seb231 I thought when we run the model we'd add a date to say this property was scored that much at that date.